docs/basic/grain_finding.md

```diff
@@ -0,0 +1,93 @@
+
+If we want to select the DNA, then we can take only the regions of the image that are above a certain height
+(threshold).
+
+Here are several thresholds to show you what happens as we increase the threshold:
+
+![height thresholds](../_static/images/grain_finding/grain_finding_grain_thresholds.png)
+
+Notice that the amount of data decreases, until we are only left with the very highest points.
+
+The aim is to choose a threshold that keeps the data you want, while removing the background and other low objects
+that you don’t want including.
+So in this example, a threshold of 0.5 would be best, since it keeps the DNA while removing the background.
+
+There are lots of objects in this mask that we don't want to analyse, but we can remove those using area thresholds in
+the next steps. These objects have been detectd because while they are small, they are still high up and above the
+background.
+
+For more information on the types of thresholding, and how to set them, see the [thresholding](thresholding.md) page.
+
+## Remove edge grains
+
+Some grains may intersect the image border. In these cases, the grain will not be able to have accuracte statistics
+calculated for it, since it is not fully in the image. Because of this, we have the option of removing grains that
+intersect the image border with the `remove_edge_intersecting_grains` flag in the config file. This simply removes
+any grains that intersect the image border.
+
+Here is a before and after example of removing edge grains:
+
+![edge grains before](../_static/images/grain_finding/grain_finding_remove_edge_grains_before.png)
+
+![edge grains after](../_static/images/grain_finding/grain_finding_remove_edge_grains_after.png)
+
+## Size thresholding
+
+In our thresholded image, you will notice that we have a lot of small grains that we do not want to analyse in our
+image. We can get rid of those with size thresholding. This is where TopoStats will remove grains based on their area,
+leaving only the right size of molecules. You will need to play around with the thresholds to get the right results.
+
+You can set the size threshold using the `absolute_area_threshold` in the config file. This sets the minimum and
+maximum area of the grains that you want to keep, in nanometers squared. Eg if you want to keep grains that are between
+10nm^2 and 100nm^2, you would set `absolute_area_threshold` to `[10, 100]`.
+
+![size thresholding before](../_static/images/grain_finding/grain_finding_size_thresholding_before.png)
+
+![size thresholding after](../_static/images/grain_finding/grain_finding_size_thresholding_after.png)
+
+## Optional: U-Net mask improvement
+
+As an additional optional step, each grain that reaches this stage can be improved by using a U-Net to mask the grain
+again. This requires a U-Net model path to be supplied in the config file.
+
+The U-Net model will take the bounding box of each grain, and predict a better mask for it, which then replaces
+the original mask.
+
```

Comment:
An example of the unet images might be good here too - there should be one from the topology paper SI figures

Reply:
Done